packages/permission-controller/src/PermissionController.test.ts

```diff
@@ -2898,7 +2887,6 @@ describe('PermissionController', () => {
 
       const callActionSpy = jest
         .spyOn(messenger, 'call')
-        // TODO: Replace `any` with type
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         .mockImplementationOnce(async (...args: any) => {
           const [, { requestData }] = args;
```

Comment:
The instances of `.mockImplementationOnce(async (...args: any) => {` in this file look like they are fixable, and not examples of intentional usage. How would you feel about restoring the TODO comments for all of these?

There's clearly an expected shape of `args` that the writer wasn't able to fully specify either because of its complexity, or because it is incompatible with `jest`'s API. In either case, this looks like an example of `any` usage that we wouldn't want to introduce into the codebase.

Reply:
That's fair. I got rid of all of them: [b7a275f](https://github.com/MetaMask/core/pull/4171/commits/b7a275f09374ddde3fe1349a2630dc1be2142add)